gantry/firmware/axis_x_hardware_config.c

```diff
@@ -21,6 +21,8 @@ GantryHardwarePin gantry_hardware_get_gpio(GantryHardwareDevice device) {
             return pinout;
 
         default:
+            pinout.port = GPIOB;
```

Comment:
we definitely want the port at least to stay null, i'd think. maybe even force a crash here.